course/src/isometry.cpp

```diff
@@ -10,9 +10,169 @@
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+namespace {
+
+Matrix3 matrix_multiplication(const Matrix3& matrix1, const Matrix3& matrix2) {
```

Comment:
This could be part of `Matrix3` class.
Makes sense to have a method for multiplying matrixes within `Matrix` class
```c++
Matrix3 product(const Matrix3& other);
```